### DISCUSSION TITLE
Improving GNSS performance

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#82] [General]
For those who have recently obtained a kit and haven't completely mounted it: If you are reasonably good at basic SMD soldering, please follow the instructions here to significantly improve your unit's GNSS precision:

1) **DO NOT** cover the transponder board with clear heat shrink tubing. It severely detunes the GPS antenna. If you have already completed that step, then carefully cut through the heat shrink to uncover the PCB. You still need the clear heatshrink to keep the lower antenna assembly together and to act as the secondary water seal. Your lower antenna assembly should look like this:

![IMG_1296](https://user-images.githubusercontent.com/1565933/134207058-53243d0b-f029-4cb3-b4ca-6a8476ec4892.JPG)

2) Use something moderately sharp to scrape a bit of solder mask to allow a "pi" shape matching network to form on the GPS antenna trace as shown below. Remove the existing capacitor and mount these components instead:

![IMG_1301](https://user-images.githubusercontent.com/1565933/134207624-bf0a87ad-03ed-4b7f-84bf-da6491bc0d12.JPG)

That's it. This will tune the antenna properly when mounted inside the PVC enclosure and improve position accuracy. It will also allow the unit to make use of GPS + GLONASS constellations.

From now on, all kits I ship will have this modification. The assembly manuals have been updated already.

**Comment by Rudder2** [comment]
Hello Peter,

I already fully assembled my unit just haven't had a chance to Mount it due
to hurricane season and medical reasons. I really want to do this fix but
think in going to have to cut apart my transponder as I heat shrinks it all
the way together line the instructions said. Do you have any advice?

Thank you for your time and help,

David

**Comment by peterantypas** [comment]
Did I send you extra heat shrink with it or not? If not, I'll just send you
some.

**Reply by peterantypas** [reply]
The main idea is to cut through each layer of heat shrink tube deep enough that you can separate it by hand, and not so deep that you end up gushing the enclosure, or the antenna tube itself or whatever is underneath it. I typically use an X-ACTO knife for this.

**Comment by Rudder2** [comment]
Hello Peter,

I just checked and no, I don't have extra shrink wrap. I never seen shrink
wrap that big before so getting more sent probably is best.

I'm so happy that there is away to upgrade the units we already bought. I
thought was going to have to purchase a new board. Thank you for the
continued engineering.

I have a UBlox GNSS on my BoatIO hat and yes, it needs an active antenna to
be accurate so your decision to stick with the current chip was good for
our application. I only bought the hat with the GNSS on it because MAIANA
wasn't ready and I needed a GPS fast. It will be nice to have a backup GPS
onboard. Things happen and redundancy it key, as I learned on my last trip
to Key West.

Thank you for all your time and awesome engineering!

David

**Comment by muebau** [comment]
How big is the chance to make it worst or even destroy the antenna part?
I have very little experiences with this small parts and I am not sure how confident I should be to try it.

**Reply by peterantypas** [reply]
I only recommend doing this if you're comfortable with this process. The detuning is not fatal, it just makes the GNSS not precise enough for critical navigation.

**Reply by muebau** [reply]
As it will become the only GPS source on our ship I need to fix it.
What size are the needed parts and does somebody have a good source in Europe?

**Reply by kagouraki** [reply]
I am also interested for the europe source. 
In Greece, you cant buy 0603 components nowhere.

**Reply by peterantypas** [reply]
Let me further qualify what I said: By "critical navigation" I mean something like negotiating a tight, unmarked opening in a reef. I haven't seen more than 6 meters of error as things stand now.

**Comment by kagouraki** [comment]
Hello. I am about to assemble the kit. 
My board is this one and as i understand i need to find a 4pf and 2.5pf capacitors and solder them in the right place. 
Is this component that my board has the 5.6nH one or i need to change that also? 
![Uploading 20211012_165322.jpg…]()

**Reply by peterantypas** [reply]
No, the board came with a 12pF series capacitor, so that needs to be replaced.

**Reply by kagouraki** [reply]
![20211012_165322](https://user-images.githubusercontent.com/6173967/136970660-315031c8-df0a-41e7-b421-891f41158e7f.jpg)

**Reply by kagouraki** [reply]
Sh!t. Where do i find such small things :/ 
Thank you peter. I will try and see if i find some nearby !

**Comment by peterantypas** [comment]
Some folks have inquired about the possibility of applying conformal coating to the PCB. I just ordered one silicon and one acrylic conformal coating spray from MG Chemicals to see how much effect they will have on the GNSS antenna. Results probably next week ...

**Comment by peterantypas** [comment]
MG Chemicals Silicone spray gets a passing grade. No appreciable effect on GNSS or AIS tuning:

![IMG_1345](https://user-images.githubusercontent.com/1565933/137647910-83fa7ba8-1ed4-4085-8898-be36684bf963.JPG)

**Reply by peterantypas** [reply]
Their acrylic spray also works fine, so either will do. I'm partial to silicon though, so that's what I'll use from now on.

**Comment by gah67** [comment]
That's very good to know - thanks for testing

**Comment by peterantypas** [comment]
Alright, this is it. We have repeatable results.
![IMG_1351](https://user-images.githubusercontent.com/1565933/137947158-2287f718-be6a-4b67-bf30-3b14b504eecd.jpg)

**Comment by matthew-quirke** [comment]
Can anyone explain how Peter checked the resonance of the GNSS antenna while it was inside the enclosure?
I have a NanoVNA, but I’m wondering about the practical side — wouldn’t soldering or terminating an RG178 or RG316 coax lead from the VNA to the microstrip leading to the antenna affect the measurement itself?

- [EDIT] - CANCEL that question I found Peter's discussion about tuning - in a reply about comparing the GNSS modules.
